recipes-browser/dinghy/dinghy_0.1.bb

```diff
@@ -0,0 +1,19 @@
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://COPYING;md5=bf1229cd7425b302d60cdb641b0ce5fb"
+DEPENDS = "wpewebkit libsoup-2.4 glib-2.0 ninja-native"
+RDEPENDS_${PN} = "libsoup-2.4-dev glib-2.0-dev wpewebkit-dev"
```

Comment:
Why does it need this extra runtime-dependencies on this dev packages?
I guess that was copied from dyz? dyz needs that because of luajit but I don't think dinghy needs this. Is a C program.